src/modules/authentication/AuthenticationWrapper.ts

```diff
@@ -50,7 +50,10 @@ export class AuthenticationWrapper implements IAuthenticationWrapper {
 
   public logOut() {
     this.deleteHomeAccountId();
-    msalApplication.logoutRedirect();
+    msalApplication.logout({
```

Comment:
The logout function is deprecated in msal. We can use the logoutRedirect function in place of this